src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -667,6 +666,8 @@ private async Task<List<ChainedHeaderBlock>> RewindToForkPointAsync(ChainedHeade
 
             disconnectedBlocks.Reverse();
 
+            this.logger.LogInformation("Reorg from block '{0}' to '{1}'", oldTip, fork);
```

Comment:
what about using string interpolation when logging?
e.g.
 this.logger.LogInformation($"Reorg from block '{oldTip}' to '{fork}'");

Reply:
We dont use string interpolation for logs because they have a perf impact, if logging is disabled the object will not  try to resolve the ToString()

Reply:
oh yes i remember now that discussion and my suggestion about `ConditionalAttribute`